pkgs/development/compilers/ghc/6.10.2-binary.nix

```diff
@@ -99,4 +99,8 @@ stdenv.mkDerivation rec {
 
   meta.license = stdenv.lib.licenses.bsd3;
   meta.platforms = ["x86_64-linux" "i686-linux"];
+
+  passthru = {
+    isGhcjs = false;
```

Comment:
It seems a bit wasteful to add attributes to every ghc expression if all but one of them are `false` anyway (and always will be). Maybe it's simpler to add an `isGhcjs = true` attribute to ghcjs only, and then refer to it as `ghc.isGhcjs or false` to cover for those expressions that don't have it?